src/components/composer/top_bar_composer/top_bar_composer.ts

```diff
@@ -23,6 +23,7 @@ css/* scss */ `
     .o-composer:empty:not(:focus)::before {
       /* svg free of use from https://uxwing.com/formula-fx-icon/ */
       content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 -10 121.8 142.9' width='24' height='24' focusable='false'%3E%3Cpath d='m28 34-4 5v2h10l-6 40c-4 22-6 28-7 30-2 2-3 3-5 3-3 0-7-2-9-4H4c-2 2-4 4-4 7s4 6 8 6 9-2 15-8c8-7 13-17 18-39l7-35 13-1 3-6H49c4-23 7-27 11-27 2 0 5 2 8 6h4c1-1 4-4 4-7 0-2-3-6-9-6-5 0-13 4-20 10-6 7-9 14-11 24h-8zm41 16c4-5 7-7 8-7s2 1 5 9l3 12c-7 11-12 17-16 17l-3-1-2-1c-3 0-6 3-6 7s3 7 7 7c6 0 12-6 22-23l3 10c3 9 6 13 10 13 5 0 11-4 18-15l-3-4c-4 6-7 8-8 8-2 0-4-3-6-10l-5-15 8-10 6-4 3 1 3 2c2 0 6-3 6-7s-2-7-6-7c-6 0-11 5-21 20l-2-6c-3-9-5-14-9-14-5 0-12 6-18 15l3 3z' fill='%23BDBDBD' /%3E%3C/svg%3E");
+      position: absolute;
```

Comment:
adding other css properties might bring other issues. Some were removed for this very reason in this commit https://github.com/odoo/o-spreadsheet/commit/9781b7e3a030e585c26117a8187712d9e986c82e

I'd go for the other option and make the svg smaller

Reply:
> adding other css properties might bring other issues. Some were removed for this very reason in this commit [9781b7e](https://github.com/odoo/o-spreadsheet/commit/9781b7e3a030e585c26117a8187712d9e986c82e)
> 
> I'd go for the other option and make the svg smaller

I tried the second approach of making a smaller SVG, but encountered a strange bug. 

When I attempted to resolve this issue within the o_spreadsheet library, it caused the composer in odoo to malfunction. Similarly, when i tried fixed the issue in odoo, it resulted in the composer in the library breaking

Therefore, I had to add some additional CSS properties.

Reply:
> adding other css properties might bring other issues. Some were removed for this very reason in this commit [9781b7e](https://github.com/odoo/o-spreadsheet/commit/9781b7e3a030e585c26117a8187712d9e986c82e)
> 
> I'd go for the other option and make the svg smaller

Regarding the conversation we had on this PR. I tried to decrease the size of the SVG in order to remove the scrollbar by setting the height and width to 16px. Unfortunately, this caused the SVG to appear smaller in the UI. 

![image](https://user-images.githubusercontent.com/121081869/234816283-015f42ac-7d8e-499e-b0b4-b929e648d490.png)


Additionally, attempting to fix the issue here creates a bug in the spreadsheet library. I would appreciate your insights on why these issues are occurring and any possible reasons for them.